asciidoctorj-core/src/main/java/org/asciidoctor/Asciidoctor.java

```diff
@@ -810,32 +810,6 @@ public static Asciidoctor create(List<String> loadPaths) {
             return JRubyAsciidoctor.create(loadPaths);
         }
 
-        /**
-         * Creates a new instance of Asciidoctor and sets a specific classloader for the  JRuby runtime to use. This method is mostly
-         * used in environments where different threads may have different classloaders, like build tools sbt or ANT.
-         *
-         * @param classloader
-         * @return Asciidoctor instance which uses JRuby to wraps Asciidoctor
-         *         Ruby calls.
-         * @deprecated Please use {@link #create()} and set the TCCL before or {@link #create(List)} passing the paths
-         * that you would have used to create the ClassLoader.
-         */
-        public static Asciidoctor create(ClassLoader classloader) {
-            return JRubyAsciidoctor.create(classloader);
```

Comment:
I think these methods can also be removed.

Reply:
Done

Reply:
@Mogztter I think we have to resurrect these methods.
I wasn't aware of this either, but it seems like these methods are needed to use AsciidoctorJ in an OSGI environment (see #721)
Until we have a replacement we should keep these methods.